packages/google_sign_in/google_sign_in/integration_test/google_sign_in_test.dart

```diff
@@ -2,8 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.9
```

Comment:
How is this passing tests? integration_test isn't null safe on stable.

Reply:
🤷  – is CI running on stable?

Reply:
I think this is passing tests, because the integration tests are not running for this package.

The location of the test file is not standard (it should live inside: google_sign_in/google_sign_in/**example**/...)

Reply:
https://github.com/flutter/flutter/issues/81929 😢